response_operations_ui/views/collection_exercise.py

```diff
@@ -159,15 +160,25 @@ def _validate_collection_instrument():
             logger.debug('Invalid file format uploaded', filename=file.filename)
             error = {
                 "section": "ciFile",
-                "header": "Error: wrong file type for Collection instrument",
+                "header": "Error: wrong file type for collection instrument",
                 "message": "Please use XLSX file only"
             }
+        else:
+            # file name format is surveyId_period_formType
+            form_type = _get_form_type(file.filename) if file.filename.count('_') == 2 else ''
```

Comment:
Put this `if` on a new line?

Reply:
Never mind ignore this comment ^^

Reply:
Feel like one line conditional statements are easier to read